content/en/docs/instrumentation/go/sampling.md

```diff
@@ -1,36 +1,45 @@
 ---
 title: Sampling
 weight: 80
-cSpell:ignore: sdktrace
 ---
 
-Sampling is a process that restricts the amount of traces that are generated by
-a system. The exact sampler you should use depends on your specific needs, but
-in general you should make a decision at the start of a trace, and allow the
+Sampling is a process that restricts the amount of spans that are generated by a
+system. The exact sampler you should use depends on your specific needs, but in
+general you should make a decision at the start of a trace, and allow the
 sampling decision to propagate to other services.
 
-A sampler needs to be set on the tracer provider when its configured, as
-follows:
+A [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) can
+be set on the tracer provider using the
+[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
+option, as follows:
 
 ```go
-provider := sdktrace.NewTracerProvider(
-    sdktrace.WithSampler(sdktrace.AlwaysSample()),
+provider := trace.NewTracerProvider(
+    trace.WithSampler(trace.AlwaysSample()),
 )
 ```
 
-`AlwaysSample` and `NeverSample` are fairly self-explanatory. Always means that
-every trace will be sampled, the converse holds as true for Never. When you're
-getting started, or in a development environment, you'll almost always want to
-use `AlwaysSample`.
+[`AlwaysSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#AlwaysSample)
+and
+[`NeverSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#NeverSample)
+are self-explanatory values. `AlwaysSample` means that every span is sampled,
+while `NeverSample` means that no span is sampled. When you're getting started,
+or in a development environment, use `AlwaysSample`.
 
 Other samplers include:
 
-- `TraceIDRatioBased`, which will sample a fraction of traces, based on the
-  fraction given to the sampler. Thus, if you set this to .5, half of traces
-  will be sampled.
-- `ParentBased`, which behaves differently based on the incoming sampling
-  decision. In general, this will sample spans that have parents that were
-  sampled, and will not sample spans whose parents were _not_ sampled.
+- [`TraceIDRatioBased`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#TraceIDRatioBased),
+  which samples a fraction of spans, based on the fraction given to the sampler.
```

Comment:
Mmmm, I think we still do want to say traces, or at least clarify what's happening here. Yes, it will sample spans, but it's really about sampling traces at  given rate at the head. For example, if you emit 100 traces each request and specify a rate of 0.5, you won't end up with 50 missing spans in each trace, but you will end up with 50% of the total span volume.

Reply:
> dif you emit 100 traces each request and specify a rate of 0.5, you won't end up with 50 missing spans in each trace

I am not sure if you are correct. I think this will be correct if you combine `TraceIDRatioBased` and `ParentBased`.

Reply:
Also see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#traceidratiobased